Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -459,3 +459,252 @@ record is selected it should simply be discarded in case shown in
 context where ordering or position matters (like in menus or column
 based page content). This is done in the appropriate places.
 
+Persistence In-Depth Scenarios
+==============================
+
+The following section represents how database records are actually persisted in a database
+table for different scenarios and previously performed actions.
+
+Placeholders
+------------
+
+Workspace placeholders are stored in field :code:`t3ver_state` which can have the following values:
+
+* `-1`: new placeholder version - the workspace pendant for a new placeholder (see value `1`)
```

Comment:
You could use a definition list. That might be a more reader friendly representation.

Reply:
Do you have an example how to do that?

Reply:
Forgot that, already had tab open: https://docs.typo3.org/m/typo3/book-extbasefluid/master/en-us/b-ExtbaseReference/Index.html#localization

This contains multiple examples for APIs, e.g. annotations with a but of code, or APIs of query builder with less code.

Reply:
Will add these changes as additional commit soon